apiml-security-common/src/main/java/org/zowe/apiml/security/common/auth/saf/SafResourceAccessConfig.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.security.common.auth.saf;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+
+@Configuration
+@Slf4j
+public class SafResourceAccessConfig {
+
+    @Bean
+    public SafResourceAccessVerifying safResourceAccessVerifying() throws IOException {
```

Comment:
could be a conditional bean? but can be done in the next PR, once another implementation is available

Reply:
I have changed it for this issue, but with the next pr it will created again with a if clausule.